.github/workflows/deploy.yml

```diff
@@ -0,0 +1,55 @@
+name: DEPLOY
+on:
+  workflow_dispatch:
+    permissions:
+      teams:
+        - 'reactor/team'
+    inputs:
+      application:
+        description: 'Application name to deploy:'
+        required: true
+        type: choice
+        default: 'projectreactor-test'
+        options:
+          - 'projectreactor-test'
+          - 'projectreactor'
+
+permissions: read-all
+
+jobs:
+  deploy-cf:
+    runs-on: reactor-self-hosted
+    env:
+      APPNAME: ${{ github.event.inputs.application }}
+      ROUTE: ${{ github.event.inputs.application == 'projectreactor' && 'https://projectreactor.io' || 'https://projectreactor-test.wdc-08-pcf1-pws-apps.oc.vmware.com' }}
```

Comment:
This [github actions ternary operator](https://docs.github.com/en/actions/learn-github-actions/expressions#example) is super confusing to me, but I suppose I get it ;)

Reply:
I get that the ternary operator can be confusing, but I've tested it locally, and it seems working for now, and given other potential issues will probably require more subsequent PRs, I'd prefer to proceed if you don't mind. Happy to revisit in a future PR if needed.
For example, maybe we could do something like this for better readability (albeit with more verbosity): 
 ```
jobs:
  deploy-cf:
    runs-on: reactor-self-hosted
    env:
      APPNAME: ${{ github.event.inputs.application }}
      ROUTE: ${{ github.event.inputs.application == 'projectreactor' && 'https://projectreactor.io' || 'https://projectreactor-test.wdc-08-pcf1-pws-apps.oc.vmware.com' }}
    steps:
      - name: Set ROUTE variable
        run: |
          if [[ "${{ github.event.inputs.application }}" == 'projectreactor' ]]; then
            echo "ROUTE=https://projectreactor.io" >> $GITHUB_ENV
          else
            echo "ROUTE=https://projectreactor-test.wdc-08-pcf1-pws-apps.oc.vmware.com" >> $GITHUB_ENV
          fi

      ...
 ```

Reply:
How about not do all this hosts public? Isn't it better to have them as enviroment variables?

Reply:
may be yes

Reply:
ok, let's do that a bit later using another PR (created #133 to not forget about that).